Services/NewsService/Utils/NewsPublisherFactory.cs

```diff
@@ -15,8 +15,8 @@ public static INewsPublisher Create(NewsPublisherTypes newsPublisherType)
         {
             switch (newsPublisherType)
             {
-                case NewsPublisherTypes.CentralBank:
-                    return new RussianCBInfo();
+                case NewsPublisherTypes.Rambler:
+                    return new RamblerRssReader();
                 default:
                     //TODO change on custom exception
```

Comment:
The time has come. Change to custom exception

Reply:
> The time has come. Change to custom exception

Changed